packages/dds/tree/api-report/tree.api.md

```diff
@@ -634,6 +634,7 @@ export class FlexObjectNodeSchema<const out Name extends string = string, const
 
 // @internal
 export interface FlexTreeContext extends ISubscribable<ForestEvents> {
+    readonly checkout: ITreeCheckout;
```

Comment:
This probably wants to go through a small API Council review. A full design probably isn't required. That said, if there will be further API changes to support validation, it might be a good idea to go through that process sooner than later.

Reply:
Not that I'm against it, but we want that for @internal interfaces? I thought about this as an implementation detail inside SharedTree.

Reply:
I agree with Alex. This shouldn't be a public API concern.